src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const bookSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.books.push(action.payload);
+    },
+    removeBook: (state, action) => {
+      const bookId = action.payload;
+      state.books = state.books.filter((book) => book.id !== bookId);
+    },
+  },
+});
+
+export default bookSlice.reducer;
```

Comment:
- [ ] You probably forgot to export the methods in the reducer. something like: `export const { addBook, removeBook } = booksSlice.actions`

Remember to apply it to the categoriesSlice too.